src/app/layouts/layout/layout-sample/layout-sample.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="sample-wrapper">
+    <section class="sample-content">
+        <article class="sample-column">
+            <div class="layout-sample" igxLayout>
```

Comment:
Do we really need all that fluff for the sample to look like it does? Please remove all markup that doesn't add any value.